content/en/building/contact-summary/contact-summary-templated.md

```diff
@@ -55,7 +55,7 @@ Each field that can be shown on a contact's profile is defined as an object in t
 | `width` | `integer` | The horizontal space for the field. Common values are 12 for full width, 6 for half width, or 3 for quarter width. Default 12. | no |
 | `translate` | `boolean` | Whether or not to translate the `value`. Defaults to false. | no |
 | `context` | `object` | When `translate: true` and `value` uses [translation variables](https://angular-translate.github.io/docs/#/guide/06_variable-replacement), this value should provide the translation variables. | no |
-| `appliesIf` | `function()` or `boolean` | Return true if the field should be shown. | no |
+| `appliesIf` | `function()` | Return `true` if the field should be shown, and `false` if it should be hidden. Default is `true`. | no |
```

Comment:
I dunno how much detail to go into.  Smells fine but slightly incomplete... it's a bit more complicated than saying "false if hidden" because visibility is determined by `appliesIf && !resolvedIf`. I think this is explained somewhat well [here](https://github.com/medic/cht-docs/blob/ab1bd5276f05ab7ee98cd09399a1513ecebb375b/content/en/building/tasks/complex-tasks.md?plain=1#L100).

Reply:
Thanks for having a look!

> visibility is determined by appliesIf && !resolvedIf

These docs are specifically for the contact-summary `fields`.  Unless I am totally missing something, I think that `resolvedIf` is just for _tasks_.  In the tasks world, yes, visibility (and everything else :sweat_smile:) is more complicated, but here for the contact summary fields, I think the logic is more straight forward.  

That being said, if I totally misunderstood what you meant, let me know and I can open a new PR to better clarify things!